autogen/agentchat/assistant_agent.py

```diff
@@ -15,9 +18,12 @@ class AssistantAgent(ConversableAgent):
     This agent doesn't execute code by default, and expects the user to execute the code.
     """
 
-    DEFAULT_SYSTEM_MESSAGE = """You are a helpful AI assistant.
+    shell_command = "sh" if sys.platform != "win32" else get_powershell_command()
+
+    DEFAULT_SYSTEM_MESSAGE = f"""You are a helpful AI assistant.
 Solve tasks using your coding and language skills.
-In the following cases, suggest python code (in a python coding block) or shell script (in a sh coding block) for the user to execute.
+For context, you are in the operating system {sys.platform}.
```

Comment:
"operating system platform"

Reply:
you want me to change it to operating system platform {sys.platform}?

Reply:
done!